integrations/tensorflow/e2e/keras/iree_vision_test_suite.oss.bzl

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 Google LLC
```

Comment:
Drop the ".oss". That means "file that is only used in oss and must be handled specially with Google's source repository". This is just a wrapper macro that should work the same in both places.

Reply:
Done.